backend.native/llvm.def

```diff
@@ -18,7 +18,8 @@ linkerOpts = -fvisibility-inlines-hidden \
     -pedantic -Wno-long-long -Wcovered-switch-default -Wnon-virtual-dtor -Wdelete-non-virtual-dtor \
     -std=c++14 \
     -DNDEBUG -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS \
-    -ldebugInfo -lcoverageMapping
+    -ldebugInfo -lcoverageMapping \
+    -Wl,-exported_symbols_list,llvm.list
```

Comment:
Not sure if this works on Linux or Windows.
Btw, do we need the similar fix on these platforms?

Reply:
I will check.